### PR TITLE
Code_of_conduct

### PR DESCRIPTION
**Pull request by gtzio**
### What does this PR do:
Add Code of conduct for the project
### Where should the reviewer start:
diffs

### CODE REVIEW

**Comment by markvincze** [comment]
:+1:

I see that the build now is failing with the alex.js test, because the conduct has some words it doesn't like.
I guess we should just add the `packages/frint/CODE_OF_CONDUCT.md` file to the `.alexignore`.

File: packages/frint/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Contributor Covenant Code of Conduct
```
Comment (fahad19) [inline packages/frint/CODE_OF_CONDUCT.md]: You can move this file to the root of the repository.

Comment (gtzio) [reply packages/frint/CODE_OF_CONDUCT.md]: thanks @markvincze and @fahad19 I will update it in a while :)

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [frintjs-conduct@googlegroups.com]
```
Comment (fahad19) [inline CODE_OF_CONDUCT.md]: The square bracket is not needed around the email address.

File: README.md [hunk 3]
```diff
@@ -6,6 +6,9 @@
 
 For documentation, visit [https://frint.js.org](https://frint.js.org).
 
+This project adheres to the Contributor Covenant code of conduct. By participating, you are expected to uphold this code.
+Please report unacceptable behavior to [frintjs-conduct@googlegroups.com].
```
Comment (fahad19) [inline README.md]: Square bracket can be removed.

**Comment by fahad19** [comment]
Thanks for the PR! Really appreciate it! 💖 

---

Note to maintainers and those who merge PRs:

* Please make sure the PR titles are descriptive enough, because they appear in the commit history once squash merged to master
* Also apply labels PR and the related semver label, which helps generate changelogs and prepare for releases later
* make sure it is assigned to someone too
* delete the branch afterwards 
* if PR is for an existing Issue, reference it with hashtag, so it is automatically closed later upon merge

The info above can be moved to MAINTAINERS.md file if it helps.

File: README.md (hunk 3 above)
Comment (gtzio) [reply README.md]: ok I will remove it tomorrow :)

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (gtzio) [reply CODE_OF_CONDUCT.md]: I will fix it tomorrow :)

**Comment by fahad19** [comment]
Closes #235

**Pull request merged**